src/handler/remove.ts

```diff
@@ -0,0 +1,10 @@
+interface RemoveOptions {
+  timeout?: number;
+}
+
+/**
+ * Remove element a document by id from the given collection
```

Comment:
Should comment read: 

`Remove element (and) document by id from the given collection`  or 
`Remove element(, a) document by id from the given collection`

Reply:
Fixed: Remove a document by id from the given collection